Brazilian Portuguese Language Support 🇧🇷 (#553)
InnerSourceCommons/innersourcecommons.org: content/pt-br/learn/books/adopting-innersource-principles-and-case-studies.md

Neste livro você pode ler sobre:

* Os princípios do "The Apache Way", nos quais o InnerSource se baseia
* Um dos primeiros projetos InnerSource conhecidos nos Bell Laboratories, cujo produto ainda é amplamente utilizado em diversos produtos
* O programa experimental InnerSource da Robert Bosch chamado BIOS, que resultou em muitos produtos inovadores
* O programa InnerSource do PayPal e como ele levou a uma redução fantástica nas interrupções dos engenheiros
* A adoção do InnerSource na Europace, uma empresa de médio porte na Alemanha que atua no domínio de serviços financeiros
* O programa InnerSource da Ericsson, que ajudou a remover gargalos em sua organização.

Por fim, fornecemos uma comparação sistemática dos cinco estudos de caso e oferecemos uma série de diretrizes para começar a adotar o InnerSource em sua organização.

## Sobre o Autor

> Apesar de ter um diploma de bacharel em literatura francesa, **Danese Cooper** tem uma carreira de 30 anos na área de tecnologia, incluindo cargos de gerenciamento sênior de engenharia e estratégia de código aberto na Apple, Microsoft, Symantec, Sun, Intel e, mais recentemente, no PayPal. Ela é uma líder reconhecida no movimento de código aberto, por seu trabalho na Sun e como CTO da Wikimedia Foundation (casa da Wikipedia), e por seus

Reviewer: ```suggestion
Apesar de ter um diploma de bacharel em literatura francesa, **Danese Cooper** tem uma carreira de 30 anos na área de tecnologia, incluindo cargos de gerenciamento sênior de engenharia e estratégia de open source na Apple, Microsoft, Symantec, Sun, Intel e, mais recentemente, no PayPal. Ela é uma líder reconhecida no movimento de código aberto, por seu trabalho na Sun e como CTO da Wikimedia Foundation (casa da Wikipedia), e por seus
```

>  muitos anos de serviço em Conselhos Diretores ou como consultora de projetos de código aberto conhecidos, incluindo a Open Source Initiative, Open Hardware Association, Mozilla, Drupal e Apache. Atualmente, ela atua como Presidente da Node.js Foundation. Danese fundou a InnerSourceCommons.org em 2015 como parte de um esforço de inicialização para o PayPal. Ela desenvolve e ministra treinamentos para organizações e indivíduos que trabalham com InnerSource por meio da DaneseWorks, sua consultoria. Ela palestra internacionalmente sobre tendências de código aberto e InnerSource, difundindo amplamente o ethos do código aberto.

Reviewer: ```suggestion
 muitos anos de serviço em Conselhos Diretores ou como consultora de projetos open source conhecidos, incluindo a Open Source Initiative, Open Hardware Association, Mozilla, Drupal e Apache. Atualmente, ela atua como Presidente da Node.js Foundation. Danese fundou a InnerSourceCommons.org em 2015 como parte de um esforço de inicialização para o PayPal. Ela desenvolve e ministra treinamentos para organizações e indivíduos que trabalham com InnerSource por meio da DaneseWorks, sua consultoria. Ela palestra internacionalmente sobre tendências de Open Source e InnerSource, difundindo amplamente a ética Open Source.
```

> Como acadêmico, **Klaas-Jan Stol** realizou pesquisas sobre código aberto e InnerSource nos últimos 10 anos. Ele é professor na Escola de Ciência da Computação e Tecnologia da Informação do University College Cork, um investigador financiado pelo Lero - o Centro de Pesquisa de Software da Irlanda, e um Investigador Principal da Science Foundation Ireland. O objetivo de sua pesquisa é entender melhor esses novos modos de trabalho de desenvolvimento e suas implicações para a indústria de software. Por exemplo, quando o código aberto estava surgindo como um tópico de pesquisa, a maioria das empresas não demonstrava muito interesse, principalmente devido a um mal-entendido sobre o que era o código aberto. Também não estava claro como as empresas poderiam se beneficiar do código aberto. Ao longo dos anos, isso mudou drasticamente devido a muitos estudos de pesquisa sobre esse tema, que são publicados em conferências acadêmicas, revistas e livros didáticos, que por sua vez são utilizados em cursos universitários. Algo semelhante está acontecendo agora com o InnerSource. Embora tenha sido um tópico de pesquisa por anos, ainda existem muitos equívocos sobre o que é o InnerSource e como as empresas podem se beneficiar ao adotá-lo. Klaas tenta preencher essa lacuna escrevendo livros como este.

Reviewer: ```suggestion
Como acadêmico, **Klaas-Jan Stol** realizou pesquisas sobre open source e InnerSource nos últimos 10 anos. Ele é professor na Escola de Ciência da Computação e Tecnologia da Informação do University College Cork, um pesquisador financiado pelo Lero - o Centro de Pesquisa de Software da Irlanda, e um Science Foundation Ireland Principal Investigator. O objetivo de sua pesquisa é entender melhor esses novos modos de trabalho de desenvolvimento e suas implicações para a indústria de software. Por exemplo, quando o open source estava surgindo como um tópico de pesquisa, a maioria das empresas não demonstrava muito interesse, principalmente devido a um mal-entendido sobre o que era o open source. Também não estava claro como as empresas poderiam se beneficiar do open source. Ao longo dos anos, isso mudou drasticamente devido a muitos estudos de pesquisa sobre esse tema, que são publicados em conferências acadêmicas, revistas e livros didáticos, que por sua vez são utilizados em cursos universitários. Algo semelhante está acontecendo agora com o InnerSource. Embora tenha sido um tópico de pesquisa por anos, ainda existem muitos equívocos sobre o que é o InnerSource e como as empresas podem se beneficiar ao adotá-lo. Klaas tenta preencher essa lacuna escrevendo livros como este.
```
